README.md

```diff
@@ -13,29 +13,48 @@ The toolchain includes the following:
 - `dojo-bind`: Generate bindings for various languages / frameworks (typescript, phaser / rust, bevy).
 
 ## Development
-### Prerequisites
-- Install [Rust](https://www.rust-lang.org/tools/install)
-- Setup Rust:
+
+### Setup Submodules
+
 ```
-rustup override set stable && rustup update && cargo test
+git submodule update --init --recursive
 ```
 
-### Using the language server
+## Development container
+
+It is recommended to use the dev container when building on DoJo as it contains everything needed to begin developing.
+
+Make sure you update your Docker to the latest stable version, sometimes the Dev containers do not play nicely with old Docker versions.
+
+Command pallete: `ctrl + shift + p`
 
-Install the cairo vscode extension: https://github.com/starkware-libs/cairo/tree/main/vscode-cairo
+Then: `Remote-Containers: Rebuild Container Without Cache`
+
+### Setup the language server 
 
-Build the Dojo language server:
 ```
+cd cairo/vscode-cairo
+
+npm install --global @vscode/vsce
+npm install
+vsce package
+code --install-extension cairo1*.vsix
+
+cd /workspaces/dojo
+
 cargo build --bin dojo-language-server --release
```

Comment:
What do you think of adding most of these steps to the `Dockerfile`?

Reply:
yes good idea - ill do that. I was doing via script, but in the container makes more sense

Reply:
The submodule setup makes this annoying in the build step. I might just leave this - as I assume this will become a VScode extension in the store very very soon. Not worth the headache right now